.github/workflows/tests-js.yml

```diff
@@ -35,3 +35,35 @@ jobs:
         run: npm ci
       - name: Run jest task
         run: npx jest ${{ matrix.suite }}/
+
+  npm_tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        suite:
+          - classy:jest
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1000
+          submodules: recursive
+      - name: Check for .nvmrc file
```

Comment:
I'd say you can remove the `.nvmrc` file check: it's always present in the repos and it has been for quite some time.